.travis.yml

```diff
@@ -1,13 +1,14 @@
-sudo: false
 language: node_js
-cache:
-  directories:
-    - ~/.npm
+cache: npm
 notifications:
   email: false
-node_js: '8'
+node_js:
+  - 10.13
+  - 12
+  - node
 install: npm install
 script: npm run validate
 after_success: kcd-scripts travis-after-success
 branches:
-  only: master
+  only:
+    - master
```

Comment:
Let's add `- next` to this

Reply:
Only in this repo or also in the template? 🤔

Reply:
In both. Sorry.